app/services/post_alerter.rb

```diff
@@ -539,6 +546,49 @@ def expand_group_mentions(groups, post)
 
   end
 
+  def expand_here_mention(post, exclude_ids: nil)
+    exclude_ids ||= []
+    user_ids = []
+
+    # users that liked posts in topic
+    if user_ids.size < SiteSetting.max_here_mentioned
```

Comment:
I think `user_ids.size` will always be `0` since it is assigned to an empty array right above.

Reply:
I preferred to have symmetric blocks for consistency.